default/static/css/custom.css

```diff
@@ -65,4 +57,27 @@
 }
 .extension-url-container .input-group {
     padding-top: 3px;
-}
\ No newline at end of file
+}
+.footer.logo {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 100px;
+    background-color: #222;
```

Comment:
@jpmckinney The CSS changes for the footer color here.

Reply:
Aha, I think it was just a browser cache issue.